tools/istio-docker.mk

```diff
@@ -91,42 +87,42 @@ docker.sidecar_injector: $(ISTIO_DOCKER)/sidecar-injector
 docker.proxy: tools/deb/envoy_bootstrap_tmpl.json
 docker.proxy: ${ISTIO_ENVOY_RELEASE_PATH}
 docker.proxy: $(ISTIO_OUT)/pilot-agent ${PROXY_JSON_FILES}
-docker.proxy: pilot/docker/Dockerfile.proxy pilot/docker/Dockerfile.proxy_debug
-	mkdir -p $(ISTIO_DOCKER)/proxy
-	cp $^ $(ISTIO_DOCKER)/proxy/
-ifeq ($(DEBUG_IMAGE),1)
-	cp ${ISTIO_ENVOY_DEBUG_PATH} $(ISTIO_DOCKER)/proxyd/envoy
-	time (cd $(ISTIO_DOCKER)/proxy && \
-		docker build -t $(HUB)/proxy:$(TAG) -f Dockerfile.proxy_debug .)
-else
-	cp ${ISTIO_ENVOY_RELEASE_PATH} $(ISTIO_DOCKER)/proxy/envoy
-	time (cd $(ISTIO_DOCKER)/proxy && \
+docker.proxy: pilot/docker/Dockerfile.proxy
+	mkdir -p $(DOCKER_BUILD_TOP)/proxy
+	# Not using $^ to avoid 2 copies of envoy
```

Comment:
as you changed the dependency, why would $^ have 2 envoy ?

Reply:
The one in deps has a sha.